crates/ruff/src/rules/pyupgrade/rules/f_strings.rs

```diff
@@ -420,7 +422,19 @@ pub(crate) fn f_strings(
     }
 
     let mut diagnostic = Diagnostic::new(FString, expr.range());
-    if checker.patch(diagnostic.kind.rule()) {
+
+    // Avoid autofix if there are comments within the call:
+    // ```
+    // "{}".format(
+    //     0,  # 0
+    // )
+    // ```
+    if checker.patch(diagnostic.kind.rule())
+        && !checker
+            .indexer()
+            .comment_ranges()
+            .intersects(TextRange::new(attr.end(), expr.end()))
```

Comment:
Instead, can you grab the `arguments` node off the call (line 321), and use `arugments.range()`? To me that better matches the semantics and is a bit more readable.

Reply:
Great suggestion. Yeah, that's much better.